Riot/Modules/Room/RoomViewController.m

```diff
@@ -1254,24 +1254,31 @@ - (void)setBubbleTableViewDisplayInTransition:(BOOL)bubbleTableViewDisplayInTran
 
 - (void)sendTextMessage:(NSString*)msgTxt
 {
+    // If the event occur on timeline not live, use the live data source to resolve event
+    BOOL isLive = self.roomDataSource.isLive;
+    if (isLive == NO) {
+        [self setupRoomDataSourceLive];
+    }
+    MXKRoomDataSource *roomDataSource = isLive ? self.roomDataSource : self.roomDataSourceLive;
```

Comment:
This is harder to think of a solution for, but as `self.roomDataSourceLive` is set in a completion handler of `setupRoomDataSourceLive`, then potentially it might not be set at this point.

Reply:
I think changes I make will fix this